phpunit/bootstrap.php

```diff
@@ -5,10 +5,12 @@
  * @package Sample_Plugin
  */
 
+// phpcs:disable
```

Comment:
Just wondering why you are using an `// phpcs:disable` here.

Why not use an `<exclude>` from the ruleset if you are disabling PHPCS for this file for all sniffs ? That way, the file won't even get tokenized.
Or you could use a ` // @phpcs:ignoreFile` in the file docblock ?
Or if you did it this way because you do want the file docblock checked, but nothing else, you could put the `phpcs:disable` at the end of the docblock to disable PHPCS for the rest of the file.

Reply:
There was too many issues to address in a suitable format - I didn't want to touch too much.

`// phpcs:disable` made most sense to me at the time, but I can update this.